src/PushPlate.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Definitions.h"
+#include "PushPlate.h"
+
+void PushPlateTriggerSystem::on_tick(void)
+{
+    for (auto&& [entity, pushplate] : access_storage().each())
+    {
+        if (pushplate.active != pushplate.activeprevious)
```

Comment:
I'd adore this line if it were something like `push_plate.active != push_plate.active_previous`, could we work on this?